source/content/guides/build-tools/01-introduction.md

```diff
@@ -23,7 +23,7 @@ multidev: true
 
 <Alert type="danger" title="Warning">
 
-The current version of the Build Tools plugin is designed for Terminus 1.x, and is not yet compatible with [current versions](/terminus/updates) of Terminus. We are working on a new version of the Terminus Build Tools plugin with Terminus 2 support and will update this guide once it is released. In the meantime, you can test the [beta release](https://github.com/pantheon-systems/terminus-build-tools-plugin/releases/latest).
+We are working on a new 2.x version of the Terminus Build Tools plugin with Terminus 2 support. In the meantime, you can test the beta release following the instructions below. If you have an issue with the build tools, review the [issue queue](https://github.com/pantheon-systems/terminus-build-tools-plugin/issues). If you need to update to the latest version of Terminus Build Tools plugin, see [updating plugin docs](https://pantheon.io/docs/terminus/plugins#update-plugins).
```

Comment:
> If you have an issue with the build tools, review the 

I think this needs to be "the build tools _plugin_" to avoid the awkward english.